debezium-microbenchmark/src/main/java/io/debezium/performance/core/FilterSmtPerf.java

```diff
@@ -5,6 +5,7 @@
  */
```

Comment:
I have been running those benchmarks on my machine, maybe the level of noise was too high, but I couldn't get reliable numbers.